pset-functions.js

```diff
@@ -206,7 +290,38 @@ Use the following test cases to confirm your program meets the success criteria
 ************/
 console.log('Problem 5:')
 
-// Add your code below this line
+//This code is not working
+//function doubleLetters(word){
+  /*let letterArr = word.split('')
+  let answer
+
+  for(let i = 0; i <= letterArr.length; i++) {
+    for(let x = 1; x <= letterArr.length; x++) {
+        if(letterArr[x] === word[i]){
+          console.log("match found")
+            answer = true
+          }
+        }
+      answer = false
+      }
+      return answer
+    }*/
+
+
+
+function doubleLetters(word){
+  let letters = word.split('')
+  let result = false
+  letters.forEach((letter, index) => {
+    if (letter === letters[index+1]){
+      result = true
+    }
+  })
+  return result
+}
+
+console.log(doubleLetters("rune"))
```

Comment:
@nmikhol Good job on this problem here. I see you split the word into an array and your looping through each element checking to see the current and next letter match, Good job on that. One suggestion instead of creating the variable result and changing it to true of false, Since your using a boolean you can return true and return result explicitly instead of changing it through a variable. since your looking for the consecutive letter in a word once you find the consecutive repeating word if you return true that would stop the program since you arrived at an answer.
Take a look at an alternate solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L291)